porch/config/deploy/3-porch-server.yaml

```diff
@@ -42,7 +42,7 @@ spec:
         - name: porch-server
           # Update image to the image of your porch apiserver build.
           image: gcr.io/example-google-project-id/porch-server:latest
-          imagePullPolicy: IfNotPresent
+          imagePullPolicy: Always
```

Comment:
this will break e2e kind tests (I think)

Reply:
I will revert this line. It changed to help my development, since I want it to always pull the newest image.